api/src/api/opportunities_v0_1/opportunity_schemas.py

```diff
@@ -191,27 +225,18 @@ class OpportunitySchema(Schema):
         }
     )
 
-    revision_number = fields.Integer(
-        metadata={
-            "description": "The current revision number of the opportunity, counting starts at 0",
-            "example": 0,
-        }
-    )
-    modified_comments = fields.String(
-        metadata={
-            "description": "Details regarding what modification was last made",
-            "example": None,
-        }
-    )
```

Comment:
Should these be removed if they're not added back anywhere?

Reply:
I removed them from the model to avoid any confusion as the summary object has similar values. While I don't know what we want to do with these long-term, the current system returns the summary values, so I removed these in the top-level opportunity for now.